tests/h/paginator_test.py

```diff
@@ -19,6 +19,38 @@ def test_current_page_defaults_to_1(self, pyramid_request):
         assert page['cur'] == 1
 
     @pytest.mark.parametrize('page_param,expected', [
+      # If the current page is the first page.
```

Comment:
👍 I like tests in this style, where test cases are essentially ASCII representations of what the UI should look like in different scenarios